collector/pkg/component/analyzer/network/protocol/testdata/rocketmq/server-trace-json.yml

```diff
@@ -0,0 +1,51 @@
+trace:
+  key: rocketmq-json
+  requests:
+    -
+      name: "sendmsg"
+      timestamp: 100000000
+      user_attributes:
+        latency: 40000
+        res: 108
+        data:
+          - "00000068000000647b22636f6465223a3130362c22666c6167223a302c226c616e6775616765223a224a415641222c226f7061717565223a313036322c2273657269616c697a655479706543757272656e74525043223a224a534f4e222c2276657273696f6e223a3339337d"
```

Comment:
Can you use ascii / hex / len|ascii format to make the data readable?

Reply:
Sorry, I don't know how to make the data more readable.Actually,It's already hexadecimal data.I will take this into consideration.

Reply:
As there is a json format in data, the json datas can be displayed as ascii.

Reply:
go it,thanks